packages/utilities/psammead-calendars/src/index.js

```diff
@@ -0,0 +1,3 @@
+import jalaali from './calendars/jalaali';
+
+export default jalaali;
```

Comment:
Probably best to do 
`export jalaali from './calendars/jalaali';` 
This will mean doing 
```
import { jalaali } from '@bbc/psammead-calendars`
```
instead of
```
import jalaali from '@bbc/psammead-calendars`
```
But will avoid breaking changes if we add a second calender type

Reply:
I agree this is more flexible, will we need a comment to silence `prefer-default-export` as well?

Reply:
Looks like it passed the es-linting with @dr3's suggestion fine.

Reply:
Haha I expected eslint to complain 🙈 odd 😛

Reply:
> Haha I expected eslint to complain 🙈 odd 😛

Maybe eslint is clever enough to detect the `export default` on the [jalaali.js](https://github.com/bbc/psammead/blob/a5e2214f6b4a318556fa879772a32a586a563812/packages/utilities/psammead-calendars/src/calendars/jalaali.js#L64) file?

Reply:
`export thing from 'thing'` isnt actually a real thing (yet), we use `@babel/plugin-proposal-export-default-from` to get the syntax, so I have a feeling eslint doesnt know what it means and just nopes out 🙈